docs/src/examples/components/Popup/Types/index.tsx

```diff
@@ -22,14 +22,9 @@ const Types = () => (
     />
     <ComponentExample
       title="Focus Trap"
-      description="Popup content traps focus on appearance by default - at the same time, this focus trap behavior can be customized."
+      description="Popup content traps focus on appearance by using accessibility behavior 'popupFocusTrapBehavior'. At the same time, this focus trap properties can be customized, by creating your own behavior (see Behaviors section for Popup component)."
```

Comment:
we should omit variable names in descriptions

Reply:
Did you mean ```popupFocusTrapBehavior```?